src/service/viewport-impl.js

```diff
@@ -1414,10 +1414,14 @@ export class ViewportBindingIosEmbedWrapper_ {
   constructor(win) {
     /** @const {!Window} */
     this.win = win;
+    const topClasses = this.win.document.documentElement.className;
+    this.win.document.documentElement.className = '';
     this.win.document.documentElement.classList.add('-amp-ios-embed');
 
     /** @private @const {!Element} */
-    this.wrapper_ = this.win.document.createElement('i-amp-html-wrapper');
+    this.wrapper_ = this.win.document.createElement('html');
```

Comment:
I am sure there is a good reason, but curious why just applying the CSS  to the existing <html> won't work and we must have a wrapper? ( two <html> elements is strange, I hope it doesn't trigger crazy edge-case bugs in Safari )